packages/canary-web-components/src/components/ic-date-picker/test/basic/ic-date-picker.e2e.ts

```diff
@@ -0,0 +1,55 @@
+import { newE2EPage } from "@stencil/core/testing";
+
+describe("ic-date-picker component", () => {
+  it("renders", async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      `<ic-date-picker label="Date picker label"></ic-date-picker>`
+    );
+
+    const element = await page.find("ic-date-picker");
+
+    expect(element).toHaveClass("hydrated");
+  });
+
+  // it.only("should open calendar when button clicked", async () => {
```

Comment:
Do we need to remove this test?

Reply:
Actually does this whole test file need removing if the e2e tests have now moved to cypress?